src/main/resources/assets/securitycraft/lang/en_us.json

```diff
@@ -428,15 +428,15 @@
 	"help.securitycraft.wire_cutters.info": "Rightclicking a mine with the wire cutters will defuse it.",
 	"help.securitycraft.keypad_item.info": "The key panel is used in the crafting recipes of keypads, password-protected chests, and password-protected furnaces.",
 	"help.securitycraft.admin_tool.info": "The admin tool's main purpose is to display information about a SecurityCraft block. It will show the owner's UUID and name, and other information, like passcodes and any inserted modules.",
-	"help.securitycraft.taser.info": "The taser is a self-recharging stun gun. Rightclick to fire it. Entities hit with the taser will get Nausea II, Slowness II, and Weakness II for 10 seconds. If the Taser is sneak-rightclicked (provided some redstone in the inventory) it will double its strength for the next shot.",
-	"help.securitycraft.camera_monitor.info": "The camera monitor is a handheld screen, used to view live camera feeds. Rightclick a security camera to view it on the monitor. Sometimes, it may automatically shutoff to save resources, simply rightclick to enable it again (configurable).",
-	"help.securitycraft.sc_manual.info": "The SecurityCraft Manual shows information about all the blocks and items SecurityCraft adds.",
+	"help.securitycraft.taser.info": "The taser is a self-recharging stun gun. Rightclick to fire it. Entities hit with the taser will get Nausea II, Slowness II, and Weakness II for 10 seconds. If the Taser is sneak-rightclicked (provided some redstone in the inventory), the potion effects for the next shot will be doubled.",
+	"help.securitycraft.camera_monitor.info": "The camera monitor is a handheld screen used to view live camera feeds. Rightclick a security camera to view it on the monitor. Sometimes, it may automatically shutoff to save resources, simply rightclick to enable it again (configurable).",
```

Comment:
Something like "The camera monitor is a portable item used to view live camera feeds. Right-click a security camera with the camera monitor to bind it, then right-click air while the camera’s chunk is loaded to see its feed. [plus that last sentence]" ?

Reply:
Perhaps `The camera monitor is a portable item used to view live camera feeds. Right-click a security camera with the camera monitor to bind it, then right-click air and select the camera in the GUI while the camera’s chunk is loaded to see its feed. You can bind multiple cameras.` (Not sure whether the last sentence here is superfluous, but it might not hurt to have it in.) The last sentence of the original info (`Sometimes, it may automatically shutoff to save resources, simply rightclick to enable it again (configurable).`) is also related to LookingGlass so it should no longer exist.

Reply:
Nice. Maybe instead of the last sentence:
- The camera monitor is a portable item used to view live feeds of up to 30 bound security cameras. [the rest of the info] Unbind a security camera by right-clicking it again or opening the camera’s monitor GUI.
- Right-click up to 30 security cameras with the camera monitor to bind them [the rest of the info]. Unbind a security camera by right-clicking it again with the camera monitor or by opening the camera’s monitor GUI.

Reply:
First one, but it should be `by opening the camera monitor's GUI and clicking the corresponding \"X\" button.` at the end ;) (the \ are there because JSON).

Reply:
Still need `camera's monitor` to be changed to `camera monitor's`.

Reply:
Good catch.